mycpp/gc_builtins.h

```diff
@@ -109,10 +107,16 @@ inline bool _cmp(Str* a, Str* b) {
   return str_cmp(a, b) < 0;
 }
 
+// TODO(Jesse): What does the following comment mean?  How does including
+// <algorithm> bloat types?  gc_heap.h already includes a ton of C++ headers so
+// I don't know how not including algorithm is a win.  Maybe this comment is
+// old..?
```

Comment:
oh I think I just meant it would bloat the ASDL translation units, not the types

In general I want to avoid too much STL code in headers, but it's not our top priority and this comment can probably be deleted

We're re-organizing the code anyway which makes it somewhat obsolete